apps/ledger-live-mobile/android/settings.gradle

```diff
@@ -6,6 +6,9 @@ project(':react-native-webview').projectDir = new File(rootProject.projectDir, '
 
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 
+include ':flipper-plugin-rn-performance-android'
+project(':flipper-plugin-rn-performance-android').projectDir = new File(rootProject.projectDir, '../node_modules/flipper-plugin-rn-performance-android')
```

Comment:
this is unrelated to the PR / what is documented in description and should be shipped separately.

Reply:
Yeah it was not my code at first, but after the rebase on LLM v3 I saw that too and wondered if I needed to clean that up.
I'll remove that